WalletWasabi.Tests/UnitTests/Transactions/TransactionProcessorTests.cs

```diff
@@ -383,36 +388,56 @@ public async Task HandlesBumpFeeAsync()
 		[Fact]
 		public async Task HandlesRbfWithLessCoinsAsync()
 		{
+			// --tx0---> (A) --+--+
+			//                 +--|-- tx2 (replaceable --+---> (myself)
+			// --tx1---> (B) --+  |                      |
+			//                    |                      +---> (change myself)
+			//                    |
+			//                    +--- tx3 (replaces tx2)
+
 			// Replaces a previous RBF transaction by a new one that contains one more input (higher fee)
 			var transactionProcessor = await CreateTransactionProcessorAsync();
 			Script NewScript(string label) => transactionProcessor.NewKey(label).P2wpkhScript;
 
 			// A confirmed segwit transaction for us
-			var tx = CreateCreditingTransaction(NewScript("A"), Money.Coins(1.0m), height: 54321);
-			transactionProcessor.Process(tx);
+			var tx0 = CreateCreditingTransaction(NewScript("A"), Money.Coins(1.0m), height: 54321);
+			transactionProcessor.Process(tx0);
 
 			// Another confirmed segwit transaction for us
-			tx = CreateCreditingTransaction(NewScript("B"), Money.Coins(1.0m), height: 54321);
-			transactionProcessor.Process(tx);
+			var tx1 = CreateCreditingTransaction(NewScript("B"), Money.Coins(1.0m), height: 54321);
+			transactionProcessor.Process(tx1);
 
 			var createdCoins = transactionProcessor.Coins.Select(x => x.GetCoin()).ToArray(); // 2 coins of 1.0 btc each
 
 			// Spend the received coins (replaceable)
 			var destinationScript = NewScript("myself");
 			var changeScript = NewScript("Change myself");
-			tx = CreateSpendingTransaction(createdCoins, destinationScript, changeScript); // spends 1.2btc
-			tx.Transaction.Inputs[0].Sequence = Sequence.OptInRBF;
-			var relevant = transactionProcessor.Process(tx);
+			var tx2 = CreateSpendingTransaction(createdCoins, destinationScript, changeScript); // spends 1.2btc
```

Comment:
Just a question: why the comment say 1.2btc why not 1.5btc is that just an example?

Reply:
Because at this moment we have 2 coins of 1btc each and the `CreateSpendingTransaction` method sends 60% to the destination and 40% to the change address so, we are sending 1.2btc and receiving 0.8 as change.